04-overall-workflow.Rmd

```diff
@@ -55,6 +55,8 @@ git push -u origin --all <repo_URL>
 
 - Work may be assigned to you as a "Work Item". In DevOps use the link to make new branch. To correct a bug, prefix the branch name with "bug/". To add new functionality, prefix the branch name with "feature/".
 
+![Some example work items in DevOps.](www/images/chapter4_work_items.png)
```

Comment:
This image doesn't seem quite right here to me. I think focussing too much on work items when this is about branches is likely to be confusing for some people.

It's perhaps slightly beyond the scope of your changes but could this section be clearer if we remove the work item content, or at least say how to make a branch without a work item too?